service/src/main.rs

```diff
@@ -478,13 +495,14 @@ fn execute_update_market<E: EnclaveBase + TeeracleApi>(
 	enclave: &E,
 ) {
 	// Get market data for usd (hardcoded)
-	let updated_extrinsic = match enclave.update_market_data_xt(node_api.genesis_hash, "usd") {
-		Err(e) => {
-			error!("{:?}", e);
-			return
-		},
-		Ok(r) => r,
-	};
+	let updated_extrinsic =
+		match enclave.update_market_data_xt(node_api.genesis_hash, "TEER", "USD") {
```

Comment:
Would be very cool to have the option to give this as an input value at start up, and not as a hardcoded strings. @brenzi should comment on that, though.

Reply:
we whitelist mrenclaves by trading pair in the teeracle pallet. therefore, we might make trading pairs configurable, yes. But then we need error handling too

Reply:
@brenzi we whitelist mrenclaves by data sources, not by trading pair. 
For now the Trading pair and the data source ("https://api.coingecko.com") are hardcoded. Both should be reworked/ updated but in another task.